pkg/olog/logrus.go

```diff
@@ -0,0 +1,359 @@
+// Copyright 2024 Outreach Corporation. All Rights Reserved.
+
+// Description: Implements a logrus.Formatter interface which follows the format used by the charm logger.
+//              Intended to be used by applications which currently use logrus and want consistently formatted
+//              while using a mix of loggers (logrus and olog). This code was mostly copied from [here](1).
+//
+//              [1]: https://github.com/charmbracelet/log/blob/82b5630d2e68c2cf4c972a926be90149fe0c60b9/text.go "Charm Text Format"
+//
+//              Example usage:
+//              ```l := logrus.New()
+//				l.SetFormatter(NewCharmTextFormatter())
+//				l.SetReportCaller(true)```
+//
```

Comment:
nit: extra spaces
```suggestion
//              ```l := logrus.New()
//		   l.SetFormatter(NewCharmTextFormatter())
//		   l.SetReportCaller(true)```
//
```

Reply:
I think some tabs ended up in there creating inconsistent spacing because it displayed fine in vscode.